src/main.go

```diff
@@ -3,6 +3,9 @@ package main
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/rs/cors"
+	"github.com/zairza-cetb/bench-routes/src/lib/filters"
+	"github.com/zairza-cetb/bench-routes/tsdb"
```

Comment:
please fix the format of importing. (same between default imports and third party imports)

Reply:
Interesting that it is not caught by the editor